src/redisearch_rs/redis_mock/src/allocator.rs

```diff
@@ -5,194 +5,137 @@
  * Licensed under your choice of the Redis Source Available License 2.0
  * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
  * GNU Affero General Public License v3 (AGPLv3).
-*/
-
-//! Alternative implementations of Redis' functions for heap allocation.
-use std::{
-    alloc::{Layout, alloc, alloc_zeroed, dealloc, realloc},
-    ffi::c_void,
-};
-
-const HEADER_SIZE: usize = std::mem::size_of::<usize>();
-const ALIGNMENT: usize = std::mem::align_of::<usize>();
-
-/// Allocates the required memory of `size` bytes for the caller usage. The caller must invoke `free_shim`
```

Comment:
Can't we keep all this documentation?

Same, for the other comment blocks below.